onnxruntime/python/_pybind_state.py

```diff
@@ -5,9 +5,23 @@
 
 import os
 import platform
+import sys
 import warnings
 import onnxruntime.capi._ld_preload  # noqa: F401
 
+# Python 3.8 (and later) on Windows doesn't search system PATH when loading DLLs,
+# so CUDA location needs to be specified explicitly.
+if platform.system() == "Windows" and sys.version_info >= (3, 8):
+    CUDA_VERSION = "10.2"
+    CUDNN_VERSION = "8"
+    cuda_env_variable = "CUDA_PATH_V" + CUDA_VERSION.replace(".", "_")
+    if cuda_env_variable not in os.environ:
+        raise ImportError(f"CUDA Toolkit {CUDA_VERSION} not installed on the machine.")
```

Comment:
You can't use f-string, we still need to support python 3.5 and this fails the pep8 check.

Reply:
flake8 (executed under Python 3.8) seems to be happy.
This could be easily made Python 3.5 compliant, but it's probably not an issue as this piece of code is executed only on Python 3.8+.
It might be time to stop supporting Python 3.5 (https://github.com/microsoft/onnxruntime/issues/5961).